### PR TITLE
Added cloning feature to autocompleteselectmultiple.change.

### PR DESCRIPTION
**Pull request by xliiv**
Updated change method on autocompleteselectmultiple widget so that it can be
used for cloning elems. This is possible by passing cloneSource in event
variable.

Pull request is not squashed because second commit is only whitespace change which would cause merging conflicts.

### CODE REVIEW

**Comment by quamilek** [comment]
Fix indentation.

File: ajax_select/static/js/ajax_select.js [hunk 1]
```diff
@@ -180,6 +180,39 @@ $.fn.autocompleteselectmultiple = function(options) {
...
+                $.each(divs, function(key, item) {
+                    var repr = $(item).children('a').html();
+                    pk = pks[key];
+                    var ui = {
+                        item: {
+                            label: pk,
+                            match: repr,
+                            pk: pk,
+                            repr: repr,
+                            url: "/assets/licences/edit/" + pk,
```
Comment (quamilek) [inline ajax_select/static/js/ajax_select.js]: Hardcoded url :anguished:

**Comment by andrzej-jankowski** [comment]
:ok:

**Comment by quamilek** [comment]
:ok:

**Pull request merged**